server/azureAppInsights.ts

```diff
@@ -31,6 +37,21 @@ function addUsernameProcessor(
   return true
 }
 
+function errorStatusCodeProcessor(
+  envelope: Contracts.EnvelopeTelemetry,
+  _contextObjects: { [name: string]: unknown } | undefined
+): boolean {
+  if (envelope.data.baseType === Contracts.TelemetryTypeString.Request) {
+    if (envelope.data.baseData !== undefined) {
```

Comment:
can this be part of the above `if` as an `&&` condition so we don't have the nested `if`?

Reply:
yes 👍